exe/krane

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "thor"
+require "kubernetes-deploy"
+
+class Krane < Thor
+  package_name "Krane"
+
+  desc "version", "Prints the version"
+  def version
+    logger.info("Krane Version: #{KubernetesDeploy::VERSION}")
+  end
+
+  private
+
+  def logger(verbose: false)
+    @logger ||= KubernetesDeploy::FormattedLogger.build(@namespace, @context, verbose_prefix: verbose)
+  end
+end
+
+Krane.start(ARGV) if __FILE__ == $PROGRAM_NAME
```

Comment:
Does this work? I manually bumped the gem and installed it locally and got this (added debug lines to check `__FILE__` and `$PROGRAM_NAME` when I noticed the version wouldn't print):
```
$ krane version
__FILE__:	/Users/timothysmith/.gem/ruby/2.3.3/gems/kubernetes-deploy-0.27.0/exe/krane
$PROGRAM_NAME: /Users/timothysmith/.gem/ruby/2.3.3/bin/krane
```

`bundle exec krane` yields the same result, whereas `bundle exec exe/krane version` behaves as expected.

Reply:
Good catch! I've re-worked the file layout,  let me know what you think.